h5p-bildetema/src/components/Button/Button.module.scss

```diff
@@ -0,0 +1,45 @@
+@use "common/abstracts" as *;
+
+.button {
+  all: unset;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  white-space: nowrap;
+  width: fit-content;
+  border-radius: 0.5rem;
+  font-family: sans-serif;
+  font-size: $font-size-14;
```

Comment:
A font-size of 14 is often too small and should mostly be used for smaller information. Think we should keep this button at 16px on mobile like the rest of the buttons on the site